src/main/java/at/hannibal2/skyhanni/features/rift/area/dreadfarm/RiftWiltedBerberisHelper.kt

```diff
@@ -122,17 +145,86 @@ object RiftWiltedBerberisHelper {
     fun onRenderWorld(event: LorenzRenderWorldEvent) {
         if (!isEnabled()) return
         if (!hasFarmingToolInHand) return
-
         if (config.onlyOnFarmland && !isOnFarmland) return
 
+        if (fallback) fallbackRender(event)
+        else primaryRender(event)
+    }
+
+    @HandleEvent
+    fun onConfigFix(event: ConfigUpdaterMigrator.ConfigFixEvent) {
+        event.move(71, "rift.area.dreadfarm.wiltedBerberis.hideparticles", "rift.area.dreadfarm.wiltedBerberis.hideParticles")
```

Comment:
you dont need to bup the version from 60 to 71, you dont change existing config elements